docs/001_develop/03_client-capabilities/023_custom-components/01_quick-start

```diff
@@ -0,0 +1,82 @@
+# Quick Start
+
+Let's create a simple web component using Genesis. There are three parts that make up a component: the HTML template, the CSS styles and the component logic. Web components can vary in complexity, from a simple button to a very detailed interactive experience.
+
+## Create Custom Element
+
+Start by importing all necessary part:
```

Comment:
Start by importing all necessary parts

Reply:
Good catch, although I will have to go over everything again as I'm sure there will be more typos and perhaps structure changes required. It's a lot